docs/UserGuide.md

```diff
@@ -23,20 +20,30 @@ Current module codes supported: CS1101S, CS1231S, CS2030, CS2040S, CS2101,
 
 1. ModuleBook3.5 can run on a computer installed with a major operating system (e.g. Windows/Mac/Linux) and Java 11.
 
-2. First, download the jar file for the latest release from [github](https://github.com/AY2021S2-CS2103T-T13-2/tp/releases)
+1. First, download the jar file for the latest release from [github](https://github.com/AY2021S2-CS2103T-T13-2/tp/releases)
 
-3. Once the jar file is ready, simply double-click the file in the download section.
+1. Once the jar file is ready, simply double-click the file in the download section.
 
-4. Refer to the [Commands](#commands) below for details of each command.
+1. Refer to the [Commands](#commands) below for details of each command.
 
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Features
+![add message](images/addCommand.png)
+1. At the top of the display is a toolbar to either exit the app (under `File`) 
+   or access this user guide (under `Help`).
+   
+1. Below the toolbar is a command line to key in any of the below commands. 
+   The result is displayed in a box underneath this command line.
+   
+1. A list of modules for which tasks are available is also shown.
+    1. For each module, the number of tasks across each workload rating is displayed.
+    1. The distribution of workload across all modules is calculated and displayed on a pie chart.
```

Comment:
are the numbers here intentionally changed to all 1's??

Reply:
If I'm not wrong, how markdown works is that the 1s here would automatically be converted to 1,2,3,4...

So it's a deliberate choice.

Reply:
> If I'm not wrong, how markdown works is that the 1s here would automatically be converted to 1,2,3,4...
> 
> So it's a deliberate choice.

Yep that's the attribute of the markdown.